wiki/Rules/hu.md

```diff
@@ -0,0 +1,79 @@
+3.  **Legyél produktív a kritikáddal, anélkül, hogy átfolyna személyes támadásba.** A kritika egy nagyszerű dolog, ha rendesen van kivitelezve, de ha csupán személyes támadást eredményezel a kifejtéseddel, akkor rosszul csinálod, és el kéne gondolkozz ezen.
+4.  **Tartsd a szálakat relevánsan, és ne élessz fel régi témákat, ha kihaltak a beszélgetések.** Ha a téma már évek óta halott, nem igazán releváns többé, hagyd azt eltűnni. Posztolj újat, vagy keress és nézd meg, hogy valaki nem-e válaszolta meg már a kérdésed előtte.
+5.  **Az avatarjaid és az aláírásaid maradjanak ésszerűek.** Ha animáltak, ne villogjanak túlságosan. Egyúttal, a fájlméretük legyen alacsony, hogy a lassabb internetcsatlakozással rendelkezők is kényelmesen tudják böngészni az oldalt. Ez ideálisan maximum fél megabyte (500 kilobyte), ahol lehetséges.
+6.  **Ne posztolj hirdetéseket, átverős oldalakat vagy bármi egyéb tolakodó tartalmat.** Az átverős vagy partner oldalak megosztása jó mókának tűnhet annak, aki posztolja, de aki rákattint annak már jóval kevésbé az, tehát ne csináld. Az automatikusan betöltődő és lejátszódó tartalmak is szintúgy beletartoznak ebbe a szabályba.
+7.  **Ismeretlen célú applikációk vagy eszközök linkelése nem engedélyezett.** Egyszerűen, ha bármi legalitási kérdés felmerül az említett applikációk kapcsán, akkor nemkívánatosak posztolásra.
+
+Beatmap Közzétételi Szabályok
+---------------------------------
+
+Ezek a szabályok szigorúan azokra a beatmap-ekre vonatkoznak amik a játékon belüli [Beatmap Közzétételi Rendszer](/wiki/Beatmap_Submission_System) használatával lettek közzétéve. Ezek a szálak az osu! kliens által automatikusan jönnek létre, amikor a [szerkesztő](/wiki/editor) segítségével a beatmap-ek publikálásra kerülnek.
+
+Léteznek jobban részletezett szabályok is a beatmap-ek felépítésére, lásd: [Rangsorolási Követelmények](/wiki/Ranking_Criteria).
+
+1. **Győződj meg arról, hogy engedélyed van mindenre, amit a beatmap-edben használsz.** Ez magába foglalja a zenéket, videókat, ütéshangokat, grafikákat, vagy bármi egyéb tartalmat, ami nem a te munkád gyümölcse. Ha nem vagy benne biztos, hogy merre kéne az efféle tartalmakat keresned, nekünk van egy folyamatosan bővülő zenegyűjteményünk a [Kiemelt Előadó](https://osu.ppy.sh/beatmaps/artists/) részlegnél, ahol az összes zene 100%-os licenccel rendelkezik az _osu!_ használatához.
+2.  **Maradj témánál a Beatmap szálakban.** Ezek a szálak/beszélgetések csupán az adott beatmap-ről szólnak, és semmi másról. Ha bármi olyan problémád van, ami nem a beatmap-hez kapcsolódik, posztold azt a helyénvaló fórumba.
+3.  **Ne plagizálj vagy próbáld ellopni mások munkáját.** Szintúgy ne töltsd fel és használd a másik munkáját az engedélyük nélkül (beleértve, de nem korlátozva, témákat és vendég nehézségeket).
```

Comment:
The "plagizálj" is a kinda unknown word for youngsters so I would avoid using that, also that and "ellopni" means the same anyway.